.github/workflows/ci.yml

```diff
@@ -24,12 +24,13 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        php: [ '7.3', '7.4', '8.0', '8.1', '8.2' ]
+        # Test against the lowest and highest PHP versions that we support.
+        php: [ '8.2', '8.3' ]
```

Comment:
I guess PHP 8.4 is broken?

Reply:
Yeah it kept failing (and fixing it would make it fail for 8.2), and as we're not going to be deploying to PHP 8.4 any time soon I figured it could wait.